scripts/run-webservice-script.sh

```diff
@@ -15,6 +15,6 @@ chmod u+x dockstore-webservice.jar
 psql -h localhost -c "create user dockstore with password 'dockstore' createdb;" -U postgres
 psql -h localhost -c "ALTER USER dockstore WITH superuser;" -U postgres
 psql -h localhost -c 'create database webservice_test with owner = dockstore;' -U postgres
-psql -h localhost -f travisci/db_dump.sql webservice_test -U postgres
+psql -h localhost -f travisci/${DB_DUMP:-db_dump.sql} webservice_test -U postgres
```

Comment:
Ideally we would get rid of the `travisci` directory, moving the files in it somewhere else -- I was confused by it at first.

Reply:
Moved to `test/` https://github.com/dockstore/dockstore-ui2/pull/1414/commits/a5bfaf92a225d281531050e9a7326bbf722c136f